### ISSUE TITLE
Improve maintainability of Case Studies

### ISSUE THREAD

**Issue by celestehorgan** [kubernetes/website#23660]
The Case Study pages are, at the moment, bespoke, brittle, and difficult to maintain. This proposal would improve the general HTML/CSS situation for these pages, making them

**CSS Inheritance and general cleanup**

- [ ] Allow case study pages to inherit from gridpage.css (or similar)
- [ ] Then remove redundant classes copy-pasted from gridpage.css. The net result should be _something_ along the lines of each case study's individual stylesheet only containing background image overrides and maybe a handful of other classes.

**HTML file maintainability**

- [ ] Ensure that all bodies of text are properly wrapped in tags (many of them are simply floating inside `<div>`s and not receiving proper styling, so wrap them in `<p>`s)
- [ ] Think about using Docsy's reusable sections instead of divs
- [ ] When possible remove inline styles.

**Bonus points**

- Consider creating a framework of reusable shortcodes that can take parameters (background colors, images, etc.) instead of a bunch of custom CSS classes, so that these pages can be composed more easily by people who don't know HTML/CSS?

/area web-development
/priority important-longterm

**Comment by habibrosyad** [comment]
I've gone ahead and attempt to address the bonus point (which is also address the other points) in #23708 😄.

**Comment by sftim** [comment]
I've checked _and removed_ “Think about using Docsy's reusable sections instead of divs” as there are now no “div” elements in the HTML source for the case studies section (English localization).

**Comment by lmktfy** [comment]
We could decide to drop the case studies page. They are not current.

/lifecycle stale
/remove-lifecycle rotten

**Comment by lmktfy** [comment]
We could instead drop the case studies from our site and change the community page to link to https://www.cncf.io/case-studies/

**Comment by Caesarsage** [comment]
> We could instead drop the case studies from our site and change the community page to link to https://www.cncf.io/case-studies/

I agree, and it could even be more specific to https://www.cncf.io/case-studies?_sft_lf-project=kubernetes

**Comment by Caesarsage** [comment]
I propose splitting this into 2 PR
1. update the link on the community page to https://www.cncf.io/case-studies?_sft_lf-project=kubernetes
2. Remove/cleanup deprecated case study folder/files (ideally this should be after Docsy upgrade)

WDYT @lmktfy  ?

**Comment by lmktfy** [comment]
Also see https://github.com/kubernetes/website/pull/56975

**Comment by lmktfy** [comment]
> I propose splitting this into 2 PR
> 
>  1. update the link on the community page to https://www.cncf.io/case-studies?_sft_lf-project=kubernetes
> 
>  2. Remove/cleanup deprecated case study folder/files (ideally this should be after Docsy upgrade)

Sounds good to me. Worth checking with SIG leadership too, though.

**Comment by lmktfy** [comment]
See https://github.com/kubernetes/website/pull/57043
